modules/ROOT/pages/to-delete-asset.adoc

```diff
@@ -1,27 +1,32 @@
-= Delete an Asset
+= Deleting an Asset
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
+Keep your asset catalog maintained by deleting outdated or redundant assets. This makes is easier for consumers to find and reuse the assets they need. 
 
-You can delete an asset in the first seven days of its creation and reuse that asset's name, group, asset ID, and version. A delete within the first seven days is known as a _hard delete_.
+When deleting assets, admins determine if the Group ID, Asset ID, and Version (GAV) information for a deleted asset can be reused for a different asset. 
 
-If you delete the asset after seven days, you can't reuse its name, business group name, account ID, and version.
-A delete after seven days is known as a _soft delete_. After seven days, if you want to reuse the asset name,
-choose a new version number or put the asset in a different business group.
+Depending on the asset, you can delete it in one of these ways:
```

Comment:
`Depending on the asset` the deletion type depends on the organization settings. It works the same for all assets